src/main/resources/l10n/JabRef_en.properties

```diff
@@ -8,8 +8,7 @@ Keep\ empty\ entries=Keep empty entries
 
 Library\ '%0'\ has\ empty\ entries.\ Do\ you\ want\ to\ delete\ them?=Library '%0' has empty entries. Do you want to delete them?
 
-Unable\ to\ monitor\ file\ changes.\ Please\ close\ files\ and\ processes\ and\ restart.\ You\ may\ encounter\ errors\ if\ you\ continue\ with\ this\ session.=Unable to monitor file changes. Please close files and processes and restart. You may encounter errors if you continue with this session.
-%0\ contains\ the\ regular\ expression\ <b>%1</b>=%0 contains the regular expression <b>%1</b>
```

Comment:
The last line seems to be odd on the right, missing linebreak?

Reply:
Should be fixed in the latest commit